### DISCUSSION TITLE
Which codec is Rustdesk using?

### DISCUSSION THREAD

**Discussion by bulvara** [rustdesk/rustdesk#539] [General]
"Some people say", that TeamViewer once contracted Ronald Bultje from the x264 project and since ever uses H.264 as codec.

Which codec does Rustdesk use?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/master/libs/scrap/src/common/codec.rs

But hardware accelerated codecs will be supported in next release 1.1.10.
